deployment.go

```diff
@@ -109,6 +110,43 @@ var deploymentStatusCmd = &cobra.Command{
 	Requires the Inertia daemon to be active on your remote - do this by 
 	running 'inertia [REMOTE] up'`,
 	Run: func(cmd *cobra.Command, args []string) {
+		config, err := client.GetProjectConfigFromDisk()
```

Comment:
I don't think we need most of this code to retrieve the remote - this is basically all already handled by:

```go
deployment, err := client.GetDeployment(strings.Split(cmd.Parent().Use, " ")[0])
// deployment has all the information for the remote
```